android/.gitignore

```diff
@@ -0,0 +1,5 @@
+/Cuberite/
+/CMakeFiles/
+/lua_native-prefix/
+/tolua_native-prefix/
+CMakeCache.txt
```

Comment:
I guess an [editorconfig](http://editorconfig.org) would take care of newlines and the such.